nerdctl/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/cybozu/golang:1.17-focal 
+
+ARG NERDCTL_VERSION=v0.20.0
+
+RUN apt-get update \
+        && apt-get install -y git make
```

Comment:
`git` and `make` are already included in the `quay.io/cybozu/golang` image and are not needed.
ref: ref: Dockerfile for ubuntu-dev, the parent of golang. (https://github.com/cybozu/ubuntu-base/blob/75eacde4f3703e031b6bfd0726468ce372123ed6/20.04/ubuntu-dev/Dockerfile#L3-L9 )

Reply:
Fixed by 0cbd0eeb9193394dd94468a37926500f42b30aff